Example/Tests/ObjectMapper/ObjectMapperPolymorphismTest.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  ObjectMapperPolymorphismTest.swift
+//  SwiftKit
+//
+//  Created by Tadeáš Kříž on 6/29/15.
+//  Copyright (c) 2015 Tadeas Kriz. All rights reserved.
+//
+
+import Quick
+import Nimble
+import SwiftKit
+
+class ObjectMapperPolymorphismTest: QuickSpec {
+
+    override func spec() {
+        describe("ObjectMapper polymorphism") {
+            let objectMapper = ObjectMapper()
+            it("serializes objects") {
+                let object1 = ChildADTO()
+                let object2 = ChildBDTO()
+                let object3 = ChildB2DTO()
+                
+                let objectArray = [object1, object2, object3]
+                let json = objectMapper.toJSONArray(objectArray)
+                debugPrintln(json)
+                
```

Comment:
These tests are not complete. We need to improve them eventually, but I'm out of time now.